tensorboard/plugins/example/setup.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import setuptools
+
+
+setuptools.setup(
+    name="tensorboard_plugin_example",
+    version="0.1.0",
+    description="Sample TensorBoard plugin.",
+    packages=["tensorboard_plugin_example"],
```

Comment:
Does this guarantee that the example.js will be part of the pip package? I don't think this would be a problem for symlink case but it may not work as intended when creating a pip package out of it.

Reply:
No, it doesn’t. Good point; we should make sure that it does.